README.md

```diff
@@ -16,3 +16,24 @@ page](https://docs.github.com/en/get-started/writing-on-github/working-with-adva
 might help with the notation for mathematical expressions.
 
 $f(n)\in o(g(n)) \iff \forall c>0, \exists n_0, \forall n\ge n_0: f(n) < c g(n)$
+
+### Little-o Proof
+
+$f(n)\in o(g(n)) \iff  \forall c>0, \exists n_0, \forall n\ge n_0: f(n) < c g(n)$
+
+$f(n)\in o(g(n)) \implies  \forall c>0, \exists n_0, \forall n\ge n_0: f(n) < c g(n)$ {bidirectional implication}
+
+$f(n)\in o(g(n)) \implies f(n)\in$ { $\forall c>0, \exists n_0, \forall n\ge n_0: f(n) < c g(n)$} {definition of belonging}
+
+$f(n)\in o(g(n)) \implies f(n)\in$ { $\forall c>0, \exists n_0, \forall n\ge n_0: f(n) \le c g(n)$} {definition of less than or equal to}
```

Comment:
I think you're missing a step here -- there are two differences between the definitions.

Reply:
The above is not the definition of O.